akka-docs/src/main/paradox/stream/operators/index.md

```diff
@@ -323,6 +323,15 @@ Operators meant for inter-operating between Akka Streams and Actors:
 |ActorSink|<a name="actorrefwithbackpressure"></a>@ref[actorRefWithBackpressure](ActorSink/actorRefWithBackpressure.md)|Sends the elements of the stream to the given @java[`ActorRef<T>`]@scala[`ActorRef[T]`] with backpressure, to be able to signal demand when the actor is ready to receive more elements.|
 |ActorFlow|<a name="ask"></a>@ref[ask](ActorFlow/ask.md)|Use the `AskPattern` to send each element as an `ask` to the target actor, and expect a reply back that will be sent further downstream.|
 
+## Compression operators
+
+Flow operators to de/compress.
```

Comment:
```suggestion
Flow operators to (de)compress.
```

Reply:
This one is generated ;-)

Reply:
Oh, but I have to generate it locally. 🤦🏼‍♂️